### ISSUE TITLE
[SPIKE] Sync with Clem to understand futures on kwenta

### ISSUE THREAD

**Issue by gberg1** [Synthetixio/issues#37]
### Summary

Objective: Fully investigate what's needed to get futures working on Kwenta

### Tasks
-  Clem will be working on the subgraphs/query logic
-  Need extra hands to work on the frontend bit

### QA Tasks
- [ ] Add first QA step here

**Comment by vbstreetz** [comment]
Closing this. Will open a new ticket once this is ready to be worked on.
